lib/wibox/widget/systray.lua

```diff
@@ -66,6 +66,10 @@ function systray:draw(context, cr, width, height)
     local cols = math.ceil(num_entries / rows)
     local bg = beautiful.bg_systray or beautiful.bg_normal or "#000000"
     local spacing = beautiful.systray_icon_spacing or 0
+    local y_offset = 0 
```

Comment:
https://github.com/awesomeWM/awesome/actions/runs/6388964492/job/17348262541?pr=3860#step:33:188

```suggestion
    local y_offset = 0
```